Content.Server/Chat/AutoEmoteComponent.cs

```diff
@@ -0,0 +1,32 @@
+namespace Content.Server.Chat;
+
+using Content.Server.Chat.Systems;
+using Content.Shared.Chat.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
+
+/// <summary>
+/// Causes an entity to automatically emote at a set interval.
+/// </summary>
+[RegisterComponent, Access(typeof(AutoEmoteSystem))]
+public sealed class AutoEmoteComponent : Component
+{
+    /// <summary>
+    /// A set of emotes that the entity will preform.
+    /// <see cref="AutoEmotePrototype"/>
+    /// </summary>
+    [DataField("emotes", customTypeSerializer: typeof(PrototypeIdHashSetSerializer<AutoEmotePrototype>)), ViewVariables(VVAccess.ReadOnly)]
+    public HashSet<string> Emotes = new HashSet<string>();
+
+    /// <summary>
+    /// A dictionary storing the time of the next emote attempt for each emote.
+    /// Uses AutoEmotePrototype IDs as keys.
+    /// <summary>
+    [ViewVariables(VVAccess.ReadOnly)]
+    public Dictionary<string, TimeSpan> EmoteTimers = new Dictionary<string, TimeSpan>();
```

Comment:
Do we need multiple emotes?

Reply:
brainfood wants cluwnes to have multiple emotes like honks, giggles, so yes I think it's needed. Also if a Cluwne turns into a zombie it's possible for it to groan while still honking and giggling.